### DISCUSSION TITLE
계획에 관한 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#90] [Q&A] [answered]
좀 더 효율적인 계획을 세우기 위한 질문입니다.

아직 애자일 관련 책을 안 읽어봐서 잘 모르지만, 멘토님께서 자주 말씀하신 계획하고 실행하기 위해서 지금 공부하고 있는 부분이나 프로젝트에 사용되는 시간, 책 읽는 시간을 생각해서 좀 더 체계적으로 관리하고 싶습니다.

하면 안되는 계획으로는 이번 주에 책 한 권 읽기 이런 계획 말고 2주에 월요일은 시간을 생각해서 책 1시간 읽기와 같은 계획을 세우고 주기적으로 실행하면서 얼마나 걸리는지, 어느 정도 읽는 지를 파악하시라는 말씀으로 이해하고 진행 중에 있습니다.

모각코는 멘토님이 만드신 스케줄에 탑승해서 매주 토요일 10반부터 12시 반까지는 공부하는 시간이니 계획적으로 그 때 마다 어떤 책을 읽어야겠다 처럼 스스로 세분화된 계획을 주 단위로 세우는게 좋을까요?

지금도 나름 하루 단위로 해야 할 일을 정리하고 실행하고 있는데 요즘엔 주 단위로 늘려서 계획하는 게 좋을 것 같다는 생각이 듭니다.

그렇다고 너무 세분화해서 다 적어 놓으면 제대로 안될 것 같은 느낌이 들기도 하고 변동성을 어떻게 고려해야 할지 감이 안 잡힙니다.

일 단위에서 주 단위로 확장하려고 하니까 감이 안잡혀서 질문드립니다..

애자일 책을 읽어보는게 빠를까요..? 

변동성을 고려할 수록 계획을 추상적으로 잡아야 할 것 같고, 구체화될 수 록 망가졌을 때 리스크가 큰 것 같습니다.

**Comment by fkdl0048** [comment]
++ 너무 급한 마음에 쓴 것 같기도 하고 내용이 많이 길게 되면 이 이야기를 온라인 멘토링때 이야기 해도 좋을 것 같습니다..!

**Comment by jongfeel** [comment, marked as answer]
지난 번에 얘기해 드렸던 것의 실천편이군요.

빠른 답변으로는 일단 계획한대로 해 보고 되돌아보고 수정한다인데
처음 계획을 잘 하고 싶으신 것 같습니다.

그러면 미리 주 단위 계획에 대해서 준비해 오시면
멘토링 때 한번 같이 검토해 보면 좋을 것 같네요.
